pkg/components/pubsub/pluggable.go

```diff
@@ -35,7 +35,7 @@ type grpcPubSub struct {
 
 // Init initializes the grpc pubsub passing out the metadata to the grpc component.
 // It also fetches and set the component features.
-func (p *grpcPubSub) Init(metadata pubsub.Metadata) error {
+func (p *grpcPubSub) Init(ctx context.Context, metadata pubsub.Metadata) error {
```

Comment:
replace the ss.Context since you are receiving as a parameter

Reply:
See above